packages/ui/src/components/Table/use-column-sort.ts

```diff
@@ -15,9 +15,10 @@ export function useColumnSort(data: Ref<TableRow[]>) {
     sortState.order = null
   }
 
-  function getNextOrder(column: TableColumnApi, order: TableColumnSortOrder) {
-    const index = column.sortOrders.indexOf(order)
-    return column.sortOrders[(index + 1) % column.sortOrders.length]
+  function getNextOrder(column: TableColumnApi, order: TableColumnSortOrder): TableColumnSortOrder {
+    const array = column.sortOrders
+    // `!` is safe because of `%` and array's non-emptiness
+    return array[(array.indexOf(order) + 1) % array.length]!
```

Comment:
Do we need restricted `sortOrders` type of  if there is `% array.length` that makes possible for user to define any sort order that he wants?

Reply:
Well, does it make sense to have `['asc', 'asc']`, or `['asc', 'desc', null, null, 'desc', 'asc']`?

Now the type of `sortOrders` allows user to have any combinations of orders **without** repetitions.

Reply:
But should **we** control it?